### PR TITLE
fixed focus_parent, moved into move_focus() function

### PR DESCRIPTION
**Pull request by taiyu-len**
fixed changing focus to parent by moving it into move_focus().

anyway, i need experience in working on actual projects so ill try to do more if i can.

### CODE REVIEW

File: sway/movement.c [hunk 1]
```diff
@@ -9,6 +9,20 @@ int move_focus(enum movement_direction direction) {
...
+			sway_log(L_DEBUG, "Focus cannot move to parent");
+			return 1;
+		} else {
+			sway_log(L_DEBUG, "Moving focus away from %p", current);
+			unfocus_all(parent);
+			focus_view (parent);
+			return 0;
+		}
+	}
+
```
Comment (SirCmpwn) [inline sway/movement.c]: Style violations - add a space between `if` and `(`, and do not include a space between the function name and `(`.

**Comment by SirCmpwn** [comment]
Looks good save for the style issues.

**Comment by taiyu-len** [comment]
no problem, just open another pr or is there some git thingy i dont know about to make small edits to current ones.

**Comment by SirCmpwn** [comment]
Just gotta push more commits to master, which is what you seem to have done. Thanks!

**Pull request merged**